src/index.js

```diff
@@ -1,6 +1,38 @@
 import './styles/styles.css';
-import showScoresTable from './modules/showScores';
-import addScoresForm from './modules/addScores';
+import showScoresTable from './showScores';
+import addScoresForm from './addScores';
+import submitScore from './modules/submitScore';
+import getAllScores from './modules/getScores';
 
 showScoresTable();
-addScoresForm();
\ No newline at end of file
+addScoresForm();
+
+const form = document.getElementById('form');
+const userNameInput = document.getElementById('user-name');
+const scoreInput = document.getElementById('user-score');
+const successTxt = document.querySelector('.success');
+const spinner = document.getElementById('spinner');
+const refreshBtn = document.getElementById('refresh-btn');
+
+getAllScores();
+
+// form.addEventListener('submit',submitScore(userNameInput.value, scoreInput.value));
```

Comment:
- [ ] You did an awesome job working on this milestone! 💯 However, you missed this commented line of code. Please remember to remove unused and commented lines since they make the code dirty and sometimes hard to read! 🤓 🔧